easybuild/tools/modules.py

```diff
@@ -747,7 +738,9 @@ def modpath_extensions_for(self, mod_names):
 
         modpath_exts = {}
         for mod_name in mod_names:
-            modtxt = self.read_module_file(mod_name)
+            if not self.exist([mod_name], skip_avail=True)[0]:
```

Comment:
yes: read_module_name() called modulefile_path() which calls get_value_from_modulefile() which also called exist() in the same way.